test/cli/spdx_json_schema_test.go

```diff
@@ -35,11 +34,6 @@ func TestSPDXJSONSchema(t *testing.T) {
 			args:       []string{"-o", "spdx-json"},
 			fixture:    imageFixture,
 		},
-		{
-			name:       "power-user:image:docker-archive:pkg-coverage",
```

Comment:
The poweruser command is no longer defaulting to spdx and instead only outputs syft-json. This test is no longer needed as we do not produce spdx anymore for this given path